src/main/java/com/hillel/items_exchange/config/SecurityConfig.java

```diff
@@ -41,25 +47,29 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .authorizeRequests()
-                .antMatchers("/",
+                .antMatchers(
+                        "/",
                         "/favicon.ico",
                         "/**/*.png",
                         "/**/*.gif",
                         "/**/*.svg",
                         "/**/*.jpg",
                         "/**/*.html",
                         "/**/*.css",
-                        "/**/*.js")
-                .permitAll()
-                .antMatchers("/v2/api-docs",
+                        "/**/*.js",
+                        "/auth/**").permitAll()
+                .antMatchers(
+                        "/v2/api-docs",
                         "/configuration/ui",
                         "/swagger-resources/**",
-                        "/swagger-ui.html")
-                .permitAll()
-                .antMatchers("/auth/**", "/adv/**", "/category/**", "/subcategory/**")
-                .permitAll()
+                        "/swagger-ui.html",
+                        "/webjars/**").permitAll()
```

Comment:
Could you explain what we receive applying this change?

Reply:
59,60 lines - removed access without authentication 
66 line - we need add this to "whitelist", for ignoring swagger in spring security for tomcat